substratevm/Reports.md

```diff
@@ -147,3 +147,16 @@ The call tree report name has the structure `call_tree_<image_name>_<date_time>.
 The object tree report name has the structure: `object_tree_<image_name>_<date_time>.txt`.
 The image name is the name of the generated image, which can be set with the `-H:Name=<name>` option.
 The `<date_time>` is in the `yyyyMMdd_HHmmss` format.
+
+#### CSV files
```

Comment:
The `Reports.md` has moved to `../docs/reference-manual/native-image/Reports.md`. Can you please rebase and add this documentation there?